authors/apps/notifications/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+from django.db.models.signals import post_save
+from authors.apps.articles.models import Article, Comment
+from ..profiles.models import Profile
+from django.dispatch import receiver
+from django.db.models import Q
+from .utils import send_emails_to_recipients
+import re
+
+
+class Notification(models.Model):
+    title = models.CharField(max_length=200)
+    body = models.TextField()
+    recipients = models.ManyToManyField(to=Profile,
+                                        related_name='notifications',
+                                        related_query_name='notification')
+    time_stamp = models.DateTimeField(auto_now_add=True)
+    read = models.BooleanField(default=False)
+
+    # def read(self, True):
```

Comment:
remove the commented lines @nyabongoedgar

Reply:
I removed the comments